sections/03-specificaties/02-component-logboek.md

```diff
@@ -28,13 +28,13 @@ De interface ***MOET*** de volgende velden implementeren:
 
 Het veld `span_id` is in implementaties voor logging.
 
-### `status_code`
+### `status`
```

Comment:
We moeten alle referenties ook updaten hiervoor: https://github.com/Logius-standaarden/logboek-dataverwerkingen/blob/f6e1d2d568d5d78279951d48efbd2a02b2729fb1/sections/03-specificaties/03-component-applicatie.md?plain=1#L19 Zoek op `status_code` in de hele repository.